geost/base.py

```diff
@@ -1110,6 +1110,65 @@ def to_qgis3d(
         qgis3d = self._create_geodataframe_3d(relative_to_vertical_reference)
         qgis3d.to_file(outfile, driver="GPKG", **kwargs)
 
+    def to_kingdom(self, outfile, vw=1500.0, vs=1600.0):
+        """
+        Write data to 2 csv files: 1) interval data and 2) time-depth chart,
+            for import in Kingdom seismic interpretation software.
+
+        Parameters
+        ----------
+        outfile : Union[str, WindowsPath]
```

Comment:
str | WindowsPath

Gebruyik van Union typing is sinds Python 3.11 niet meer nodig